bluesky_queueserver/server/server.py

```diff
@@ -9,6 +9,11 @@
 logger = logging.getLogger(__name__)
 
 
+# Login and authentication are not implemented, but some API methods require
+#   login data. So for now we set up fixed user name and group
+_login_data = {"user": "John Doe", "user_group": "admin"}
```

Comment:
Can these parameters be a part of the kwargs for the `__init__` method below? Otherwise, it's hard to pass an arbitraty name if needed.

Reply:
The name and user group should be determined at the time of user login. Since login is not implemented in the demo, we will use fixed user and group names.